spec/javascript/e2e/SupplementalApplicationPage/confirmation_modal.e2e.js

```diff
@@ -53,18 +53,13 @@ describe('SupplementalApplicationPage confirm modal', () => {
       expect(url).toBe(
         `http://localhost:3000/lease-ups/applications/${LEASE_UP_LISTING_APPLICATION_ID}`
       )
-
-      await testBrowser.close()
```

Comment:
this was just a minor fix to use the functionality that we have to only setup + login to one browser session per file.